interfaces/remoteOperatorUI/index.js

```diff
@@ -79,9 +80,78 @@ function startHTTPServer(localUIApp, port) {
     const http = require('http').Server(localUIApp.app);
     const io = require('socket.io')(http);
 
+    const objectsPath = path.join(os.homedir(), 'Documents', 'spatialToolbox');
```

Comment:
nit: this can be wrong depending on which objectsPath server.js settles on

Reply:
Good point, didn't think of that. I made a PR to the server to add an API for this:
https://github.com/ptcrealitylab/vuforia-spatial-edge-server/pull/509